### PR TITLE
travis: Switch to go 1.11

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mpatel@redhat.com>

### CODE REVIEW

File: lib/container_server.go [hunk 1]
```diff
@@ -193,7 +193,7 @@ func New(ctx context.Context, config *Config) (*ContainerServer, error) {
 		if missingHookDirectoryFatal || !os.IsNotExist(err) {
 			return nil, err
 		}
-		logrus.Warnf("failed to load hooks: {}", err)
```
Comment (runcom) [inline lib/container_server.go]: so nice, that's rust :smile:

Comment (mrunalp) [reply lib/container_server.go]: Yeah, go 1.11 tests caught it :)

Comment (runcom) [reply lib/container_server.go]: rust's better!!!!!!!!!!

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.10
+FROM golang:1.11
```
Comment (mrunalp) [inline Dockerfile]: Looks like final 1.11 tagged golang image is not present yet.

File: lib/container_server.go (hunk 1 above)
Comment (mrunalp) [reply lib/container_server.go]: No doubt ;)

**Comment by runcom** [comment]
`golang:1.11` image has been pushed to docker hub and should be available now, we can fix the `Dockerfile` as well (https://hub.docker.com/r/library/golang/tags/)

**Comment by rhatdan** [comment]
LGTM

**Comment by mrunalp** [comment]
@runcom Update to go1.11 in Dockerfile.

**Pull request merged**